styles2.css

```diff
@@ -145,10 +149,260 @@ section.action {
 
 button {
   border: none;
+}
+
+button:hover {
+  background: #36b37f;
+  box-shadow: 0 8px 16px rgba(54, 179, 127, 0.24);
+  border-radius: 4px;
+}
+
+button:focus {
+  background: #008552;
+  border-radius: 4px;
+}
+
+button:enabled {
+  background: #36b37f;
+  border-radius: 4px;
+}
+
+section.headline > button {
   margin-top: 58px;
 }
 
-@media only screen and (min-width: 768px) {
+/*
+Second part of project
+*/
+section.recentWork {
+  margin-top: 10%;
+  width: 100vw;
+  height: auto;
+  padding: 64px 24px;
+  display: grid;
+  grid-template-columns: 1fr;
+  align-items: center;
+  gap: 58px;
+}
+
+h2 {
+  text-align: center;
+  font-weight: 600;
+  font-size: 24px;
+  line-height: 32px;
+  color: var(--letter-colorRC);
+  padding: 0 auto;
+}
+
+div.detail > header > h2 {
+  margin: 0 auto;
+}
+
+.bar {
+  width: 48px;
+  height: 4px;
+  background-color: var(--dark-green);
+  border-radius: 24px;
+  margin: 0 auto;
+}
+
+section.recentWork > article {
+  width: 90vw;
+  height: 90vh;
+  border-radius: 8px;
+  background-color: var(--dark-white);
+  border: 1px solid #d0d9d4;
+}
+
+section.recentWork > article > div {
+  width: 100%;
+  height: 50%;
+}
+
+section.recentWork > article > div.detail {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 16px;
+  gap: 4%;
+  background: #fff;
+  border-radius: 8px;
+}
+
+/*
+section.recentWork > article > div.detail > ul
+*/
+ul.language {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  flex-wrap: wrap;
+  padding: 0;
+  gap: 8px;
+  width: 90%;
+}
+
+ul.language > li {
+  height: 32px;
+  padding: 8px 12px;
+  background-color: var(--dark-white);
+  border-radius: 4px;
+}
+
+.buttonCol > span {
+  padding: 12px;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  color: white;
+  letter-spacing: 0.03em;
+}
+
+ul.language > li > span {
+  font-weight: 600;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  color: var(--letter-colorRC);
+}
+
+.buttonCol {
+  background-color: var(--dark-green);
+  width: 40%;
+  height: 48px;
+  min-width: 122px;
+  max-width: 200px;
+  border-radius: 8px;
+  margin-left: 30%;
+  margin-right: 30%;
+}
+
+/*
+button state
+*/
+
+section.aboutMe {
+  width: 100vw;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 64px 24px;
+  gap: 58px;
+}
+
+section.aboutMe > article.text {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 0;
+  gap: 24px;
+}
+
+article.text > header {
+  width: 100%;
+}
+
+article.text > header > h2 {
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  color: var(--text-color1);
+  margin: 0 auto;
+}
+
+article.text > button {
+  margin-bottom: 0;
```

Comment:
- [ ] There are a couple of things you need to fix in this section.
- The images on the bottom left and top right are missing. please fix that 👍
- The positioning of the button and alignment of its inner text needs a little adjustment. kindly treat it 💯 
- The font of the `p` tag should tally with the Figma design   

# Your present design :
![image](https://user-images.githubusercontent.com/50990925/212448003-9df54efd-9411-4040-94bb-5bf2127763d9.png)

# Figma design:
![image](https://user-images.githubusercontent.com/50990925/212448112-ee8e00dd-248c-4565-a3e2-2e310c38f6e5.png)